<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QTabWidget" name="open_file_tables">
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="file_manage">
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>文件资源管理器</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="FileTreeWidget" name="files_tree_manage">
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="preview_manage">
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="FileBrowser" name="file_preview_text">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="status_bar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>22</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_meun">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="open_file_action"/>
    <addaction name="open_json_action"/>
    <addaction name="open_folder_action"/>
    <addaction name="separator"/>
    <addaction name="export_checked_files_action"/>
    <addaction name="export_all_files_action"/>
    <addaction name="separator"/>
    <addaction name="export_checked_json_action"/>
    <addaction name="export_now_json_action"/>
    <addaction name="export_all_json_action"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="edit_menu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="undo_action"/>
    <addaction name="resume_aciton"/>
    <addaction name="separator"/>
    <addaction name="show_preview_action"/>
    <addaction name="show_source_action"/>
    <addaction name="show_hideen_data_action"/>
    <addaction name="show_not_tran_action"/>
    <addaction name="separator"/>
    <addaction name="replace_action"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="file_meun"/>
   <addaction name="edit_menu"/>
  </widget>
  <action name="open_file_action">
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="open_json_action">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>打开JSON</string>
   </property>
  </action>
  <action name="open_folder_action">
   <property name="text">
    <string>打开文件夹</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="export_checked_files_action">
   <property name="text">
    <string>导出选中文件</string>
   </property>
  </action>
  <action name="export_all_files_action">
   <property name="text">
    <string>导出全部文件</string>
   </property>
  </action>
  <action name="export_checked_json_action">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>导出选中的词汇</string>
   </property>
  </action>
  <action name="export_now_json_action">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>导出当前全部词汇</string>
   </property>
  </action>
  <action name="export_all_json_action">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>导出数据库全部词汇</string>
   </property>
  </action>
  <action name="show_preview_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示预览</string>
   </property>
  </action>
  <action name="show_hideen_data_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>只显示过滤字符串</string>
   </property>
  </action>
  <action name="show_not_tran_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>只显示未修改字符串</string>
   </property>
  </action>
  <action name="show_same_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>只显示重复字符串</string>
   </property>
  </action>
  <action name="undo_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>撤回</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="resume_aciton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>恢复</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="replace_action">
   <property name="text">
    <string>批量替换</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="show_source_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示原字符串</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>./Component/filetreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>FileBrowser</class>
   <extends>QPlainTextEdit</extends>
   <header>./Component/filebrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
